### ISSUE TITLE
Font formatting variations aren't rendered properly in the comments

### ISSUE THREAD

**Issue by SecularSteve** [TeamPiped/Piped#1555]
### Describe the feature

The text formatting is falsely rendered in Piped, when special formatting in-text commands should make them look different instead of like the others.

### Why would this be useful to add?

It would give a bit more life to both Piped, and the comments, if the text is just a tad bit more unique instead of completely plain through and through.

Might even make the context from some other users in some cases a bit more clear and creative.

### Concept(s)

What I typed in:

```
*bold*
_italic_
-strikethrough-
```

What it looks like in YouTube:
![Screenshot 2022-10-09 at 22-56-50 (1) Junior Jack - My Feeling (Slowed Down_Improved) - YouTube](https://user-images.githubusercontent.com/33793273/194779668-ebc6a986-ebaa-4f4a-8dca-c4f2ecd02369.png)

What it looks like in Piped:
![Screenshot 2022-10-09 at 23-03-27 Piped](https://user-images.githubusercontent.com/33793273/194779674-d7a76e49-19f8-4216-9fa5-3c20fcdb8ba5.png)

### Additional context

This was tested with my own account.

### Acknowledgements

- [X] I have searched the existing issues and this is **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this form.

**Comment by FireMasterK** [comment]
Can you give an example video for this?

**Comment by SecularSteve** [comment]
> Can you give an example video for this?

Is a video really necessary for something like this?
https://www.howtogeek.com/797411/strike-through-bold-italics-text-in-youtube-comment/

**Comment by FireMasterK** [comment]
> Is a video really necessary for something like this?
> https://www.howtogeek.com/797411/strike-through-bold-italics-text-in-youtube-comment/

Yes, a tutorial on how to make a comment isn't helpful, as I don't have an account to create such comments myself.

**Comment by SecularSteve** [comment]
> Yes, a tutorial on how to make a comment isn't helpful, as I don't have an account to create such comments myself.

https://piped.kavin.rocks/watch?v=Qip1VkTuM70
I hope it's good enough at least.
The point here is:
All of the YT comments viewed in Piped that are typed and posted in this way aren't shown like this. They're all just plain, as evidenced in the 2nd photo above.

If you have any more questions, feel free to ask.

**Comment by vanduc1102** [comment]
Hi, 
VideoId: `Qip1VkTuM70`
Comment from API is plain text, eg: "I like how this was straight to the point. Unlike other channels lol Thank you!"

I think we need to fix the issue from API first, then this ticke
